copier/main.py

```diff
@@ -60,6 +60,9 @@
 _T = TypeVar("_T")
 
 
+Operation = Literal["copy", "recopy", "update"]
```

Comment:
ˋrecopyˋ is just a shortcut to ˋcopyˋ from the same source as the last time. Maybe ˋrecopyˋ should be removed from this list?

Ping @copier-org/maintainers

Reply:
I don't have a preference here. Left it as-is for now since most applications will just care about `update` vs everything else, but I'm fine with `recopy` == `copy` as well.

Reply:
I think `copy` & `update` should be enough for now because `recopy` is just `copy` with some minor usage convenience. If it turns out later that distinguishing `copy` and `recopy` makes sense, we can easily add `recopy`. But unless there's a clear use case for `recopy`, I'd not add it prematurely.

Reply:
Alright, I removed `recopy` now. :)